.changeset/thirty-glasses-shave.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/cli': major
+---
+
+Replace sample domain with IANA-reserved domain for technical documentation
```

Comment:
definitely not a major change :)
Please modify this to be a `patch` change

Reply:
Yeah I was going to ask about this. Is it sufficient to change `major` to `patch`, or are there other actions I need to perform to make sure that the version number changes accordingly?

Reply:
just replacing `major` to `patch` is enough :)

Reply:
Thank you! Done.